wordcloud.py

```diff
@@ -57,41 +55,41 @@ def process_string(text_to_process):
 
 def determine_occurrences(list_data):
     """Add list to dictionary with value based on occurrence."""
-    word_to_occurrence = {}
+    word_to_count = {}
     for word in list_data:
-        word_to_occurrence[word] = list_data.count(word)
-    return word_to_occurrence
+        word_to_count[word] = list_data.count(word)
+    return word_to_count
 
 
 def allocate_values(list_data):
     """Add list to dictionary with value based on nominated keywords."""
-    word_to_occurrence = {}
+    word_to_count = {}
     for word in list_data:
-        word_to_occurrence[word] = 1
+        word_to_count[word] = 1
         if word in WORDS_WITH_VALUE:
-            word_to_occurrence[word] = (list_data.count(word) * 5)
-    return word_to_occurrence
+            word_to_count[word] = (list_data.count(word) * 5)
```

Comment:
The idea came from Lindsay's comment on his lecture word cloud.

What if he had wanted to know which Python keywords he mentioned most often in the lecture?  Unfortunately, the words that showed most prominently were words such as "one", "question", "like" & "know".  However, what if he wanted to know whether he mentioned "lists", "dictionaries" or "classes" most often?  It's kind of filtering out the extraneous words and I thought it was a useful way to find occurrences of keywords within a larger dataset.

Reply:
Ah I see, so you want to _filter_ the words so that only words we specify appear in the word cloud. Perhaps instead of making this a model this could be a function: maybe called `filter_words`.

Here is some sample code for how we might use a filter_words function, and what the expected output would be:
```
words = ["words", "words", "to", "filter"]
words_to_keep = ["words", "filter", "dog"]
filtered_words = filter_words(words, words_to_keep)
print(filtered_words)  # Expected output:  ["words", "words", "filter"]

word_to_count = map_word_to_occurrence(filtered_words)
print(word_to_count)  # Expected output:  {"words": 2, "filter": 1}
```

Reply:
Maybe...  But, I was still keen to have all words left in the cloud.  I applied a value of 1 to all, but extra to the keywords.  I feel that's still an interesting way to view it (if that's OK with you).